core/src/main/java/com/github/kaktushose/jda/commands/dispatching/reply/ConfigurableReply.java

```diff
@@ -118,32 +151,70 @@ public ConfigurableReply editReply(boolean editReply) {
     /// @return the current instance for fluent interface
     @NotNull
     public ConfigurableReply keepComponents(boolean keepComponents) {
-        this.keepComponents = keepComponents;
+        replyAction.keepComponents(keepComponents);
         return this;
     }
 
     /// Whether to keep the selections of a string select menu when sending edits. This setting only has an effect with
     /// [#keepComponents(boolean)] `true`.
     @NotNull
     public ConfigurableReply keepSelections(boolean keepSelections) {
-        this.keepSelections = keepSelections;
+        replyAction.keepSelections(keepSelections);
         return this;
     }
 
     /// Access the underlying [MessageCreateBuilder] for configuration steps not covered by [ConfigurableReply].
     ///
+    /// @param builder the [MessageCreateBuilder] callback
+    /// @return the current instance for fluent interface
+    /// @implNote This method exposes the internal [MessageCreateBuilder] used by JDA-Commands. Modifying fields that
+    /// are also manipulated by the Reply API, like content or embeds, may lead to unexpected behaviour.
+    ///
     /// ## Example:
     /// ```
     /// event.with().builder(builder -> builder.setFiles(myFile)).reply("Hello World!");
-    /// ```
-    ///
-    /// @param builder the [MessageCreateBuilder] callback
-    /// @return the current instance for fluent interface
+    ///```
     public ConfigurableReply builder(Consumer<MessageCreateBuilder> builder) {
-        builder.accept(this.builder);
+        replyAction.builder(builder);
         return this;
     }
 
+    /// Acknowledgement of this event with a text message.
+    ///
+    /// @param message the message to send or the localization key
+    /// @param placeholder the placeholders to use to perform localization, see [I18n#localize(Locale , String, I18n.Entry...) ]
+    /// @return the [Message] that got created
+    /// @implSpec Internally this method must call [RestAction#complete()], thus the [Message] object can get
+    /// returned directly.
+    ///
+    /// This might throw [RuntimeException]s if JDA fails to send the message.
+    public Message reply(@NotNull String message, I18n.Entry... placeholder) {
+        return replyAction.reply(message, placeholder);
+    }
+
+    @NotNull
+    public SendableReply embeds(String... embeds) {
+        return embeds(Arrays.stream(embeds).map(it -> this.embeds.get(it, event.getUserLocale().toLocale())).toArray(Embed[]::new));
+    }
+
+    @NotNull
+    public SendableReply embeds(Embed... embeds) {
+        replyAction.addEmbeds(Arrays.stream(embeds).map(Embed::build).toArray(MessageEmbed[]::new));
+        return new SendableReply(this);
+    }
+
+    @NotNull
+    public SendableReply embeds(String embed, Consumer<Embed> consumer) {
+        return embeds(embeds.get(embed, event.getUserLocale().toLocale()), consumer);
+    }
+
+    @NotNull
+    public SendableReply embeds(Embed embed, Consumer<Embed> consumer) {
```

Comment:
I think adding `embeds(Embed, I18n.Entry)` for direct placeholder support without lambda would be useful, as it's the most used of all configuration options for embeds imo.